### PR TITLE
support upgrade feature

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot Parab <prajot.parab@ibm.com>

### CODE REVIEW

**Comment by Prajyot-Parab** [comment]
@yussufsh @bpradipt Do try this out (whenever feasible) and let me know your thoughts.

File: openshift-install-powervs [hunk 1]
```diff
@@ -31,6 +31,7 @@ Available commands:
   variables       Interactive way to populate the variables file
   create          Create an OpenShift cluster
   destroy         Destroy an OpenShift cluster
+  upgrade         Upgrade an OpenShift cluster
```
Comment (bpradipt) [inline openshift-install-powervs:34]: Why we need to do upgrade using this script and not by simply following the available instructions and using `oc` or the web ui ?

**Comment by bpradipt** [comment]
@Prajyot-Parab I think we don't need to add the upgrade feature to the script. Let it be performed via OC cli or ui.
Are you fine closing this PR ? or still think we should add upgrade feature to this helper script ?

**Pull request closed without merging**